webui/react/src/components/PageHeaderFoldable.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button, Dropdown, Menu } from 'antd';
+import React, { CSSProperties, useState } from 'react';
+
+import Icon from 'components/Icon';
+
+import css from './PageHeaderFoldable.module.scss';
+
+export interface Option {
+  icon?: React.ReactNode,
+  isLoading?: boolean,
+  key: string;
+  label: string;
+  onClick?: () => void;
+  tooltip?: string;
+}
+
+interface Props {
+  foldableContent?: React.ReactNode,
+  leftContent: React.ReactNode,
+  options?: Option[],
+  style?: CSSProperties;
+}
+
+// const wrapWithTooltip =
```

Comment:
we have 1 case (trial with no checkpoint) where we would need to show a tooltip on buttons.

We have 2 issues with this:

tooltip don't work too fine on mobile
<Menu> doesn't work fine if <Menu.Item> are wrapped in <Tooltip>

Possible alternative solutions are to hide the btn or to make it clickable (not disabled) and open a modal to show the error msg. Other ideas?

Reply:
I think if the tooltip is inside the `Menu.Item` but around the label text it should work? Not 100% sure, I believe `NavigationSideBar` has a working example of it?

Reply:
This works but is not so solid.

The tooltip is triggered only when focusing the text inside the Menu.Item, not the whole Menu.Item.